src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
             }
           }
         },
+        // TODO: Should this be privateMedicalRecordsRelease or privateRecordsRelease? 
+        privateMedicalRecordsRelease: {
```

Comment:
This whole page is just [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/7421), right?

Reply:
That is correct.